cloud/shared/bin/setup.py

```diff
@@ -31,6 +32,31 @@ def run(config: ConfigLoader, params: List[str]):
     log_args = f"\"{image_tag}\" {current_user}"
 
     try:
+        resources = template_setup.detect_backend_state_resources()
+        if resources['bucket'] or resources['table']:
+            msg = inspect.cleandoc(
+                """
+                ERROR: Terraform backend state resources already exist. You may destroy these resources
+                and recreate them, but you must ensure there are no other deployed resources present.
+                Verify this by checking the AWS console for the presence of any civiform resources.
+                If there are, by recreating the Terraform backend state files, Terraform will lose track 
```

Comment:
This sentence is still a little confusing to me "If there are, by recreating the Terraform backend state files, Terraform will lose track of those deployed resources, and subsequent deploys will fail due to the resources already existing."

Would this be correct? "If additional resources are present, Terraform will lose track of them when the backend state files are recreated, and subsequent deploys will fail due to the resources already existing